go.mod

```diff
@@ -5,21 +5,22 @@ go 1.20
 require (
 	github.com/Microsoft/go-winio v0.5.2
 	github.com/containerd/continuity v0.4.1
-	github.com/gogo/protobuf v1.3.2
 	github.com/moby/patternmatcher v0.5.0
 	github.com/opencontainers/go-digest v1.0.0
 	github.com/pkg/errors v0.9.1
+	github.com/planetscale/vtprotobuf v0.6.0
```

Comment:
do we want to separate vtproto as well here related to https://github.com/moby/buildkit/pull/5342#issuecomment-2371804240?